addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2198,8 +2201,20 @@ const ListUserValueWidget = UserValueWidget.extend({
     /**
      * @private
      */
-    _onListItemChange() {
-        this._notifyCurrentState();
+    _onListItemBlur(ev) {
+        if (!this.el.contains(ev.relatedTarget)) {
+            // Update the target and the editor.
+            this._notifyCurrentState(false);
+        }
+    },
+    /**
+     * @private
+     */
+    _onListItemInput() {
+        if (!this.el.dataset.noPreview || this.el.dataset.noPreview === 'false') {
```

Comment:
I don't think other widget check the data-no-preview stuff? Or I am wrong? I thought it was an option's config, not an UserValueWidget's one (maybe it should but then it would require changing all the other widgets ?)